raiden-ts/src/services/epics.ts

```diff
@@ -389,27 +389,10 @@ export const pathFindServiceEpic = (
   );
 };
 
-function channelEntries(channels: RaidenState['channels']): ReadonlyArray<[string, Channel]> {
```

Comment:
So with the new channel structure this is not needed anymore, because channels contain all the needed information?

Reply:
Exactly.